Commivoyajer_User_Interface/Form1.cs

```diff
@@ -132,20 +135,33 @@ private void greedyMethodButton_Click(object sender, EventArgs e)
             }
 
             var input = _prepareData.PrepareData(coords);
+
+            var stopWatch = new Stopwatch();
+            stopWatch.Start();
+            var result = _greedyMethod.GetGreedyMethod(input);
+            stopWatch.Stop();
+
+            result.CalculationTime = stopWatch.ElapsedMilliseconds;
+            result.JourneyLength = _prepareData.CalculateJourneyLength(input, result.Sequence);
+
+            ShowDataInUI(coords, result);
         }
 
         private void ShowDataInUI(List<Input> coords, Output output)
         {
+            var sequenceString = "";
```

Comment:
Use StringBuilder here. It's not a TypeScript

Reply:
lazy